photon-core/src/main/java/org/photonvision/vision/frame/consumer/MJPGFrameConsumer.java

```diff
@@ -135,20 +135,14 @@ private static String videoModeToString(VideoMode mode) {
     }
 
     private static String pixelFormatToString(PixelFormat pixelFormat) {
-        switch (pixelFormat) {
-            case kMJPEG:
-                return "MJPEG";
-            case kYUYV:
-                return "YUYV";
-            case kRGB565:
-                return "RGB565";
-            case kBGR:
-                return "BGR";
-            case kGray:
-                return "Gray";
-            default:
-                return "Unknown";
-        }
+        return switch (pixelFormat) {
+            case kMJPEG -> "MJPEG";
+            case kYUYV -> "YUYV";
+            case kRGB565 -> "RGB565";
+            case kBGR -> "BGR";
+            case kGray -> "Gray";
+            case kUYVY, kUnknown, kY16 -> "Unknown";
```

Comment:
Part of the value of Java 17 is these kinds of situations bubble up. Is this the expected behavior?

Reply:
wdym by bubble up?

Reply:
Are kUYY and kY16 really 'unknown'? Or is this behavior actually a bug nobody caught because of the old syntax?

Reply:
Probably a bug I'd add a todo and open an issue to support them